### ISSUE TITLE
Accessibility - Umbraco Cloud - Profile Settings - There is no focus on buttons when tabbing through content.

### ISSUE THREAD

**Issue by Truealex4** [umbraco/Umbraco-CMS#13338]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.3.0 of Cloud

### Bug summary

When I attempt to tab through the content on the Profile Settings page within Umbraco Cloud, there are several buttons that do not come into focus when the are selected.

### Specifics

https://www.s1.umbraco.io/profile

### Steps to reproduce

**STEPS:**

1. Login to the CMS with a valid account
2. Select 'Profile' from the dropdown in top right of the screen where you see your username
3. Tab through the content using keyboard only

![image](https://user-images.githubusercontent.com/98405431/199204363-4eb4699d-c961-4936-9ac9-1fafd097b78d.png)

### Expected result / actual result

**EXPECTED RESULT:** 
All buttons, links that acre active should come into focus in the same way that they would if you were hovering over them with a mouse.

**ACTUAL RESULT:** 
The following buttons (CTA's) do not have any focus at all:

- Site logo (although there is a line after it)
- Create New Project
- Edit Profile
- Change Password
- Chat icon
